### PR TITLE
Adjust `process.command*` to be opt-in

### PR DESCRIPTION
**Pull request by inssein**
Fixes https://github.com/open-telemetry/semantic-conventions/issues/626

## Changes

I made it match the `container` [resource](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/resource/container.md).

## Merge requirement checklist

* [x] [CONTRIBUTING.md](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md) guidelines followed.
* [x] Change log entry added, according to the guidelines in [When to add a changelog entry](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#when-to-add-a-changelog-entry).
  * If your PR does not need a change log, start the PR title with `[chore]`
* [ ] [schema-next.yaml](https://github.com/open-telemetry/semantic-conventions/blob/main/schema-next.yaml) updated with changes to existing conventions.

### CODE REVIEW

**Review by lmolkova — APPROVED** [review]
I believe command_args attributes should be defined as template attributes and have per-value opt-in mechanism similar to http headers. Created https://github.com/open-telemetry/semantic-conventions/issues/790.
It can be done separately from this PR.

**Comment by trisch-me** [comment]
@inssein we do have a new way of creating a [changelog](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#adding-a-changelog-entry). Please update your PR

**Comment by trisch-me** [comment]
Should we also add notes to the properties about sensitive content as well as we do for URL for example?
https://github.com/open-telemetry/semantic-conventions/blob/main/model/registry/url.yaml#L63

**Comment by joaopgrassi** [comment]
I'd just want to hear from @breedx-nr and his comment on https://github.com/open-telemetry/semantic-conventions/issues/626#issuecomment-1972085449 before we merge this.

**Comment by joaopgrassi** [comment]
I took the liberty to fix the CI issues. Had to generate the markdown tables and add a change log. I think this is a breaking change. 

CC @open-telemetry/semconv-system-approvers

**Comment by inssein** [comment]
Ah I totally missed the earlier notification, thank you @joaopgrassi for doing that.

**Comment by Oberon00** [comment]
Note that this makes the conventions stricter. Before, if you set one of the `process.command*`, you didn't need to set `process.executable*`. Now you do. My understanding of these "at least one of" groups is that if you set one of them then all the others are non-required (although it's undefined whether they are then opt-in or recommended).

So I'm not sure that the change does what you want it to do. `process.command*` was more or less opt-in already.

File: .chloggen/ISSUE-626.yaml
Comment (trisch-me) [inline .chloggen/ISSUE-626.yaml:2]: it should be only process here, we don't have resource in registry

**Comment by trisch-me** [comment]
@inssein please reopen PR again if you still want to implement this feature

**Comment by rogercoll** [comment]
@inssein Would you be interested in reopening this PR? The PR was closed by the github-actions bot as being inactive, this PR is very helpful as would solve https://github.com/open-telemetry/semantic-conventions/issues/626

**Pull request closed without merging**
